myco_api_client/cli.py

```diff
@@ -87,6 +89,7 @@ def run(base_setup_path, setup_module_name,
         web_socket if web_socket else websocket_domain_name_from_env())
     os.environ["MYCO_CLIENT_SIMULATION_ID"] = (
         simulation_id if simulation_id else simulation_id_from_env())
+    os.environ["MYCO_CLIENT_RUN_ON_REDIS"] = "true" if run_on_redis else "false"
```

Comment:
Why not using the same pattern as `MYCO_CLIENT_WEBSOCKET_DOMAIN_NAME ` ie having a helper function to get default settings like `run_on_redis_from_env `

Reply:
We could add such method for sure, but I don't think it's really worth it
the `MYCO_CLIENT_WEBSOCKET_DOMAIN_NAME` has it's own function because it will get the value from a constant which can be changed
However, the `MYCO_CLIENT_RUN_ON_REDIS` accepts either true or false, this is why I suppose we can keep the assignment here